libafl/Cargo.toml

```diff
@@ -56,6 +56,8 @@ fork = ["libafl_bolts/derive"]
 ## Collected stats to decide if observers must be serialized or not (which should reduce mem use and increase speed)
 adaptive_serialization = []
 
+## If this feature is set, `LibAFL` targets (and the fuzzer) will crash on `SIGPIPE` on unix systems.
+handle_sigpipe = []
```

Comment:
should be default?

Reply:
Why?

Reply:
I don't think in normal fuzzing you ever want to catch sigpipe

Reply:
nvm i read it wrong